### PR TITLE
Changing all paths to use the relative path.

### PR DESCRIPTION
**Pull request by battletoilet**
Added a task gulp --production to generate the website with the production paths.

### CODE REVIEW

**Comment by mikewheaton** [comment]
Approved 🎉

File: gulp/Documentation.js [hunk 1]
```diff
@@ -12,7 +12,12 @@ var folderList = Utilities.getFolders(Config.paths.componentsPath);
 var demoPagesList = Utilities.getFolders(Config.paths.srcDocsJSCompPages);
 var reload = require('require-reload')(require);
 var BuildConfig = require('./modules/BuildConfig');
+var production = process.argv.indexOf('--production') >= 0;
+var relativePath = "/";
 
+if (production) {
+    relavtivePath = "/fabric-js/";
```
Comment (Jahnp) [inline gulp/Documentation.js]: Typo--this might cause the path to not switch

File: gulp/Documentation.js [hunk 2]
```diff
@@ -12,7 +12,12 @@ var folderList = Utilities.getFolders(Config.paths.componentsPath);
 var demoPagesList = Utilities.getFolders(Config.paths.srcDocsJSCompPages);
 var reload = require('require-reload')(require);
 var BuildConfig = require('./modules/BuildConfig');
+var production = process.argv.indexOf('--production') >= 0;
```
Comment (Jahnp) [inline gulp/Documentation.js]: This is super minor, but [`office-ui-fabric-react` uses `isProduction` for this flag](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/gulpfile.js#L10).

File: gulp/Documentation.js (hunk 1 above)
Comment (battletoilet) [inline gulp/Documentation.js]: Good catch!

**Comment by Linda-Editor** [comment]
Microsoft no longer supports this content and will not be responding to bugs or issues. We recommend that you use the newer version, [Office UI Fabric](https://developer.microsoft.com/fabric), with React as your front-end framework. We are closing this issue; if you still need assistance with Fabric.js, visit [Stack Overflow/office-ui-fabric](https://stackoverflow.com/questions/tagged/office-ui-fabric).

**Pull request closed without merging**